x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/event_filters_flyout.tsx

```diff
@@ -173,6 +175,8 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
         onClose={handleOnClose}
         data-test-subj="eventFiltersCreateFlyout"
         {...flyoutProps}
+        // EUI TODO: This z-index override of EuiOverlayMask is a workaround, and ideally should be resolved with a cleaner UI/UX flow long-term
```

Comment:
This component (`EventFiltersFlyout`) already support setting the `maskProps` _(which if I remember correctly was added to make adjustment for timeline)_, so there is no need to make this change here.  Can we instead move this fix to the Timeline code that invokes this flyout and set `maskProps` there?

Searching for usages of it, I see the following two places where this component is used:

- `x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx:330`, and
- `x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/footer.tsx:168`

I'm assuming one of these is the one you need to specifically set the `maskProps` on.

Reply:
good call @paul-tavares. I've implemented what you suggested on the `footer.tsx` component which works. I'm not familiar with the `alert_context_menu.tsx` code and don't know where in the UI it's accessible from. I can trace back to a usage in this [getDefaultControlColumns](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/timeline/body/control_columns/index.tsx) code, but in the UI I can't find anywhere we would open this flyout from?